in_toto/runlib.py

```diff
@@ -389,8 +328,15 @@ def in_toto_run(name, material_list, product_list,
   products_dict = record_artifacts_as_dict(product_list)
 
   log.info("Creating link metadata...")
-  link = create_link_metadata(name, materials_dict, products_dict,
-      link_cmd_args, byproducts, return_value)
+  link_dict = {
+    "name" : name,
+    "materials" : materials_dict,
+    "products" : products_dict,
+    "command" : link_cmd_args,
+    "byproducts" : byproducts,
+    "return_value" : return_value
+  }
+  link = in_toto.models.link.Link.read(link_dict)
```

Comment:
wouldn't it be more convenient to just use the constructor with all the parameters from above here?